samples/net/google_iot_device/src/example_utils.c

```diff
@@ -17,6 +17,7 @@ extern iotc_crypto_key_data_t iotc_connect_private_key_data;
 static iotc_timed_task_handle_t delayed_publish_task =
     IOTC_INVALID_TIMED_TASK_HANDLE;
 
+#if 0
```

Comment:
These lines can be completely removed from here.

Reply:
> These lines can be completely removed from here.

Ack!